Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/InstructionsWithImage/InstructionsWithImageViewController.swift

```diff
@@ -29,6 +29,18 @@ public final class InstructionsWithImageViewController: UIViewController {
         super.init(nibName: "InstructionsWithImage", bundle: .module)
     }
     
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: false)
+
+        if viewModel.rightBarButtonTitle != nil {
+            self.navigationItem.rightBarButtonItem = .init(title: viewModel.rightBarButtonTitle?.value,
+                                                           style: .done,
+                                                           target: self,
+                                                           action: #selector(dismissScreen))
+        }
+    }
+    
```

Comment:
Not necessarily a comment specifically on this PR, but we are repeating this block of code on almost every `GDSCommon` view controller. Feels like we should consider sub classing `UIViewController` or adding an extension. Would be interesting to discuss further at some point.